providers/provider-protocol-reference.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Provider Protocol Reference
+category: 63d7e8bdbf7b4b0e0745e823
```

Comment:
```suggestion
category: 63e0c7cf7166bf006d3b4117
```

This will list the Provider Protocol Reference under the [Reference tab](https://fiberplane-docs.readme.io/reference) in the docs

Reply:
Sure!